pkg/repositories/transformers/execution.go

```diff
@@ -119,6 +124,7 @@ func CreateExecutionModel(input CreateExecutionModelInput) (*models.Execution, e
 		User:                  requestSpec.Metadata.Principal,
 		State:                 &activeExecution,
 		LaunchEntity:          strings.ToLower(input.LaunchEntity.String()),
+		Tags:                  tags,
```

Comment:
is it worth putting the writes behind a feature flag?

Reply:
I feel like it's unnecessary because we won't add a new row to the `tags` table if users don't pass any tags to the execution Spec.